commands/utils/utils.go

```diff
@@ -186,3 +225,25 @@ func IsMajorVersionChange(fixVersionCandidate string, currVersionStr string) boo
 	currentMajorVersion := strings.Split(currVersionStr, separator)[index]
 	return candidateMajorVersion != currentMajorVersion
 }
+
+func ValidateBranchName(branchName string) error {
+	if len(branchName) == 0 {
+		return nil
+	}
+	invalidChars := regexp.MustCompile(branchNameRegex)
```

Comment:
As I suggest before, `MustCompile` should berun only once. You can therefore have a private const that is initialized with `regexp.MustCompile(branchNameRegex)`.